src/Request.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE DataKinds #-}
+
+module Request (done, view, add, delete, reset) where
+-- TODO: maybe use qualified import?
+import Network.HTTP.Req
+    ( responseStatusCode,
+      port,
+      ignoreResponse,
+      req,
+      useHttpURI,
+      defaultHttpConfig,
+      runReq,
+      ReqBodyJson(..),
+      HttpMethod, jsonResponse, responseBody, HttpBodyAllowed, AllowsBody, CanHaveBody (CanHaveBody))
+import Text.URI (mkURI)
+import Data.Maybe (fromJust)
+import Data.Text (concat, pack)
+import Prelude hiding (concat)
+import Data.Aeson (Value, ToJSON)
+import Data.Kind (Type)
+import Control.Monad.IO.Class (MonadIO)
+import GHC.IO (liftIO)
+import Relude (lookupEnv, toText, fromMaybe, readMaybe)
+
+data Response = Response {
+   message :: Value
+  ,responseCode :: Int
```

Comment:
I suppose we would run ormolu in latter PRs.

Reply:
Yes, I will be running them.